utils/redis_plan.csv

```diff
@@ -0,0 +1,35 @@
+,service_name,plan_name,mem_limit,conn_limit,cost
+0,heroku-redis,heroku-redis:hobby-dev,25,20,0
+1,heroku-redis,heroku-redis:premium-0,50,40,15
+2,heroku-redis,heroku-redis:premium-1,100,80,30
+3,heroku-redis,heroku-redis:premium-2,250,200,60
+4,heroku-redis,heroku-redis:premium-3,500,400,120
+5,heroku-redis,heroku-redis:premium-5,1000,1000,200
+6,heroku-redis,heroku-redis:premium-7,7000,10000,750
+7,heroku-redis,heroku-redis:private-7,7000,10000,900
+8,heroku-redis,heroku-redis:premium-9,10000,25000,1450
+9,heroku-redis,heroku-redis:private-9,10000,25000,1750
+10,heroku-redis,heroku-redis:premium-10,25000,40000,3500
+11,heroku-redis,heroku-redis:private-10,25000,40000,4000
+12,heroku-redis,heroku-redis:premium-12,50000,65000,6500
+13,heroku-redis,heroku-redis:private-12,50000,65000,7500
```

Comment:
Is this CSV needed?

Reply:
It was there in master too, I updated it to create plan entries for Azure. We can remove it because the database is already updated with plan entries.

Reply:
okay. The git diff is not working on the CSVs. Hence, got confused